ios/Tests/AboutFeatureTests/AboutFeatureTests.swift

```diff
@@ -3,6 +3,10 @@ import TestUtils
 import XCTest
 
 final class AboutFeatureTests: XCTestCase {
+    override func setUp() {
```

Comment:
It seems better to create custom XCTestCase class to make sure we don't have to add this everytime

Reply:
I don't think either will change.
When using custom XCTestCase, it is necessary to inherit so as not to forget.